dataland-frontend/tests/e2e/fixtures/lksg/LksgDataFixtures.ts

```diff
@@ -50,19 +65,61 @@ export function generateProductionSite(undefinedProbability = 0.5): LksgProducti
  * @returns 0 to 5 random production sites
  */
 export function generateArrayOfProductionSites(undefinedProbability = 0.5): LksgProductionSite[] {
-  return Array.from({ length: faker.number.int({ min: 0, max: 5 }) }, () =>
-    generateProductionSite(undefinedProbability)
+  return generateArray(() => generateProductionSite(undefinedProbability));
+}
+
+/**
+ * Generates a random product
+ * @returns a random product
+ */
+function generateProduct(): LksgProduct {
+  return {
+    productName: faker.commerce.productName(),
+    productionSteps: valueOrUndefined(generateArray(() => faker.commerce.productName())),
+    relatedCorporateSupplyChain: valueOrUndefined(faker.commerce.productName()),
```

Comment:
We can do better/more realisitc here I think.

I suggest we make the productionStep fake data a little longer to better represent real data.
Maybe just something like     "Put in a ${faker.commerce.productMaterial()}" or we could even fake the verb to pick a random one out of a selection (e.g. "put in a", "set the", "integrate all")

And for the related `relatedCorporateSupplyChain` we could also use the productMaterial() faker.   What do you think?

Reply:
Then we have a similar string everywhere. I don't like constance in test data. However we could do that choosing from multiple such sentences.

Reply:
Well then we can fake a word + a material:

`faker.lorem.word() + faker.commerce.productMaterial()` for `productionSteps`

And my suggestion for the `relatedCorporateSupplyChain` gets along without any constance

Reply:
For `relatedCorporateSupplyChain` we could even use `faker.lorem.sentence()`.
This fakes a whole sentence!

Reply:
Uuh. Ok. Let's do this.

Reply:
Hmmm. lorem benutzt halt wirklich nur lateinische Wörter :)

Reply:
faker.word ist aber fine. Pushe gleich.

Reply:
Hmm. faker.word.verb erzeugt bei mir deutsche Verben. Dafür sind dann aber Perlen wie "verjüngen Luxurious Cotton Chicken" dabei. Ich hab das so gepusht.

Reply:
😆 😆 .         Ok lateinisch ist blöd. Das ziehe ich dann zurück.

Kann es sein dass das mit den deutschen Verben daran liegt, dass wir den faker auf Deutsch benutzen?  Die Städte und Firmennamen sind ja auch alle sehr deutsch.

Sollten wir das nicht vllt einfach mal auf Englisch stellen?   Ist ja kein großer change, aber eigentlich sinnvoll.

Was meinst du?

Reply:
Einen ganzen lateinischen Satz als dummy finde ich ok. Das deutsch englisch gemisch stört mich persönlich nicht. Ich kann aber verstehen, wenn das jemand problematisch sieht. Können wir gerne umstellen.

Reply:
Machen wir

Reply:
Taten wir

Reply:
Love it.   Note that we should revert the switch to English in our faker, if the CI throws up tons of new problems.  It wouldn't be worth it in that case.